From 48ba1bc5c09dc169dedb0fdcb1e148d7efca1594 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Tue, 24 Sep 2024 07:22:55 +0300
Subject: [PATCH] [FSW-9949] fs1: spi flash offset change

with specs h15 version 1.19, spi flash offset
changed. Hence, those offset changes are taken
care as part of this patch.

Change-Id: I5c03bb3ddbdc3e29e7eb3674c32a107d48292ff1
---

diff --git a/arch/fs1/arc/soc.h b/arch/fs1/arc/soc.h
index a880052..b908140 100644
--- a/arch/fs1/arc/soc.h
+++ b/arch/fs1/arc/soc.h
@@ -61,7 +61,7 @@
 #define HL_FW_MEM_SIZE			SZ_4MB
 
 #define HL_GEN_MEM_BASE			REGION_BASE_ADDR(REGION_GEN_MEM_IDX)
-#define HL_FLASH_OFFSET			0x4000000
+#define HL_FLASH_OFFSET			0xF000000
 #define HL_FLASH_SIZE			(16 * SIZE_MB)
 
 /* Below definitions are applicable to IO Die 0 only */
